src/routes/give-away/create.js

```diff
@@ -0,0 +1,51 @@
+import models from '../../models'
+
+export default {
+
+  method: 'post',
+  path: '/giveaways?',
+
+  requireAuthentication: true,
+
+  handler(request, response) {
+
+    // @TODO: This is getting claned up by the orphan job
+    return models.CodexRecordFile.findById('5b1ef57078d795dc2bae3560')
+      .lean()
+      .then((leanMainImage) => {
+        delete leanMainImage._id
+
+        return models.CodexRecordFile.create(leanMainImage)
+      })
+      .then((giveawayMainImage) => {
+        const giveawayMetadata = {
+          creatorAddress: '0x0', // placeholder because it's required
```

Comment:
Maybe use `ethUtil.zeroAddress()` here instead so it's padded out to the right number of zeros? (This should probably also be done in `src/routes/test/create-images.js` but since that's just test data and won't ever be permanent it's less of a concern.)